conventions/src/main/kotlin/io.opentelemetry.instrumentation.base.gradle.kts

```diff
@@ -119,3 +119,32 @@ if (testLatestDeps) {
     }
   }
 }
+
+tasks {
+  val generateInstrumentationVersionFile by registering {
+    val name = computeInstrumentationName()
```

Comment:
I think we need to add this name to inputs too since it's not in the output dir name.